exasol_integration_test_docker_environment/lib/test_environment/create_ssl_certificates_task.py

```diff
@@ -0,0 +1,124 @@
+
+    def create_docker_volume(self) -> DockerVolumeInfo:
+        self.remove_volume(self.volume_name)
+        with self._get_docker_client() as docker_client:
+            volume = docker_client.volumes.create(
+                name=self.volume_name,
+            )
+            volume_info = self.get_volume_info(reused=False)
+        return volume_info
+
+    def remove_volume(self, volume_name):
+        try:
+            with self._get_docker_client() as docker_client:
+                docker_client.volumes.get(volume_name).remove()
+                self.logger.info("Removed volume %s", volume_name)
+        except docker.errors.NotFound:
+            pass
+
+    def cleanup_task(self, success):
+        if (success and not self.no_cleanup_after_success) or \
+                (not success and not self.no_cleanup_after_failure):
+            try:
+                self.logger.info(f"Cleaning up volume %s:", self.volume_name)
+                self.remove_volume(self.volume_name)
+            except Exception as e:
+                self.logger.error(f"Error during removing volume %s: %s:", self.volume_name, e)
+
+    def create_certificate(self) -> None:
+        template_str = pkg_resources.resource_string(
+            "exasol_integration_test_docker_environment",
+            "docker_db_config/create_certificate.sh")  # type: bytes
+        template = Template(template_str.decode("utf-8"))
+        rendered_template = template.render(#HOST_NAME='db_container_exasol_test.db_network_exasol_test',
+                                            HOST_NAME=f"{self.db_container_name}.{self.network_name}",
+                                            cert_dir=CERTIFICATES_MOUNT_PATH)
+        test_container_image_info = \
+            self.get_values_from_futures(self.test_container_image_future)["test-container"]  # type:
+
+        volumes = {
+            self.volume_info.volume_name: {
+                "bind": CERTIFICATES_MOUNT_PATH,
+                "mode": "rw"
+            }
+        }
+
+        with self._get_docker_client() as docker_client:
+            try:
+                test_container = \
+                    docker_client.containers.create(
+                        image=test_container_image_info.get_target_complete_name(),
+                        name=self.test_container_name,
+                        network_mode=None,
+                        command="sleep infinity",
+                        detach=True,
+                        volumes=volumes,
+                        labels={"test_environment_name": self.environment_name, "container_type": "test_container"},
+                        runtime=self.docker_runtime
+                    )
+                test_container.start()
+                docker_container_copy = DockerContainerCopy(test_container)
+                docker_container_copy.add_string_to_file("scripts/create_certificates.sh", rendered_template)
```

Comment:
The original file is `create_certificate.sh`, without s

Reply:
Renamed the original script to create_certificates.sh, as this is the correct name.